clio-server/config/docker-compose.yml.ctmpl

```diff
@@ -13,6 +17,22 @@ app:
     - /etc/localtime:/etc/localtime:ro
     - {{$app_dir}}/{{$app_conf}}:{{$conf_dir}}/{{$app_conf}}:ro
     - {{$app_dir}}/{{$logback_conf}}:{{$conf_dir}}/{{$logback_conf}}:ro
-    - /local/clio_logs:{{env "CLIO_LOG_DIR"}}
   restart: always
+  links:
+    - logger
+  depends_on:
+    - logger
+  volumes_from:
+    - logger
+
+logger:
+  image: broadinstitute/fluentd-gcp:latest
+  expose:
+    - 5140/tcp
+    - 5140/udp
```

Comment:
I'll remove them for now.. and if things don't work we can revisit. No point in exposing ports that don't need to be exposed.